build.gradle

```diff
@@ -27,6 +27,9 @@ ext.kafkaVersion = project.kafkaVersion
 ext.awaitalityVersion = "3.1.6"
 ext.avroVersion = "1.9.2"
 ext.puppycrawlCheckstyleVersion = "8.18"
+ext.stdlibFilepathVersion = project.stdlibFilepathVersion
+ext.stdlibIoVersion = project.stdlibIoVersion
+ext.stdlibSystemVersion = project.stdlibSystemVersion
 
```

Comment:
The build passed without the other standard library versions here? 🤔

Reply:
@BuddhiWathsala

Reply:
Yes. Just noticed this. It passed somehow.

Reply:
Yeah. that's what!

Reply:
I also wonder, how it got resolved dependencies at the first phase.

Reply:
Maybe from Ballerina-lang? Anyways, I added those dependencies back.